src/xdocs/config_javadoc.xml

```diff
@@ -812,10 +812,10 @@ public int checkReturnTag(final int aTagIndex,
     </section>
 
     <section name="JavadocParagraph">
+      <p>Since Checkstyle 6.0</p>
       <subsection name="Description" id="JavadocParagraph_Description">
-        <p>Since Checkstyle 6.0</p>
         <p>
-          Checks that:
+          Checks the Javadoc paragraph. Checks that:
```

Comment:
please make it `Checks Javadoc paragraphs be separated by empty line.`
To make sure that this Check is not responsible for all kind of validations over paragraph, as second validation is optional we can skip it from summary.

Reply:
done